Gemfile.lock

```diff
@@ -266,47 +266,48 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
+    bootsnap (1.6.0)
```

Comment:
### [bootsnap](https://github.com/Shopify/bootsnap)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.4.5 → 1.6.0

[_[change-log](https://github.com/Shopify/bootsnap/blob/master/CHANGELOG.md), [source-code](https://github.com/Shopify/bootsnap), [gem-diff](https://my.diffend.io/gems/bootsnap/1.4.5/1.6.0)_]





<details>
<summary>Commits</summary>

A change of **81** commits. See the full changes on [the compare page](https://github.com/Shopify/bootsnap/compare/v1.4.5...v1.6.0).

These are the first 10 commits:
- (f15ce20) [Remove a temporary variable](https://github.com/Shopify/bootsnap/commit/f15ce208ce2aa911e100f1fb9658b2e6f586920a)
- (826c0d6) [Add const qualifiers to the parameters and remove unsafe poi…](https://github.com/Shopify/bootsnap/commit/826c0d61dbd60db62f604b97cb9915656ecc4567)
- (2a3644c) [Only strip elidable extensions to make short forms](https://github.com/Shopify/bootsnap/commit/2a3644ce35a325c8427d7079c1e19080fe0596d5)
- (065f909) [Merge pull request #274 from Ryooooooga/remove-unsafe-pointe…](https://github.com/Shopify/bootsnap/commit/065f909a0e0fa05ccc4e26d5ede3bcc655779020)
- (ecdd65e) [Only strip elidable extensions to make short forms (#271)](https://github.com/Shopify/bootsnap/commit/ecdd65ea370024ad855bdfb5b6a096b48783c343)
- (b720530) [Update minimum ruby version and fix rubocop](https://github.com/Shopify/bootsnap/commit/b7205305841a792064ffcc4e4a5e127ff688ee37)
- (78be2f5) [Merge pull request #280 from alebruck/update-ruby-version](https://github.com/Shopify/bootsnap/commit/78be2f5aec196d8d4a72f21ba03ad8fd41adf276)
- (8d4b28a) [Don't break when Pathname is passed to LoadedFeaturesIndex](https://github.com/Shopify/bootsnap/commit/8d4b28ac609ddb0c613e41bba0cd428e591cc69f)
- (9278a85) [Don't break when Pathname is passed to LoadedFeaturesIndex (…](https://github.com/Shopify/bootsnap/commit/9278a85ffba5590c782ec7f5577064ec689b8ddf)
- (70bef22) [Update documentation on os_version -> ruby_platform](https://github.com/Shopify/bootsnap/commit/70bef22bc0b86436384afe6340ce8a68b9e3cd0f)


</details>